### PR TITLE
S05_US35: Added price listé

### PR DESCRIPTION
**Pull request by Erdnaxela45**

### CODE REVIEW

**Review by maxime-guichard — APPROVED** [review]

File: templates/home/price_list.html.twig [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            <div class="row pb-3 d-flex justify-content-center">
+                <div class=" col-lg-3 data-card">
+                    <h2 class="pb-2 text-center d-flex align-items-center justify-content-center">Poster une annonce</h2>
+                    <h3 class="post mt-2 text-center">
+                        120.00
+                        <i class="fas fa-euro-sign"></i>
+                    </h3>
+                    <p>Cette offre vous donne accès au fait de pouvoir 1 annonce. Ajoutez votre pierre à l'édifice en permettant aux étudiants de prendre connaissance de votre entreprise.</p>
+                    <div class="d-flex justify-content-center pb-3">
+                        <a href="#" class="postuler btn btn-light text-center">Payer</a>
```
Comment (LucyMerlier) [inline templates/home/price_list.html.twig:21]: "payer"??? when are you going to add a payment process to your website?

File: templates/home/price_list.html.twig [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        <div class="container">
+            <h3 class="text-center pt-3 my-5">Grille tarifaire</h3>
+            <div class="row pb-3 d-flex justify-content-center">
+                <div class=" col-lg-3 data-card">
+                    <h2 class="pb-2 text-center d-flex align-items-center justify-content-center">Poster une annonce</h2>
+                    <h3 class="post mt-2 text-center">
+                        120.00
+                        <i class="fas fa-euro-sign"></i>
+                    </h3>
+                    <p>Cette offre vous donne accès au fait de pouvoir 1 annonce. Ajoutez votre pierre à l'édifice en permettant aux étudiants de prendre connaissance de votre entreprise.</p>
```
Comment (LucyMerlier) [inline templates/home/price_list.html.twig:19]: "au fait de pouvoir POSTER une annonce"

File: templates/home/price_list.html.twig [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            </div>
+            <h2 class="m-3 d-flex justify-content-center pt-4 pb-5">Recruter le profil idéal pour votre entreprise</h2>
+            <div class="row">
+                <div class="col-12 col-lg-7">
+                    <div>
+                        <ul class="list-unstyled">
+                            <li>Vous cherchez à recruter ? Premiers emplois, stages, alternances.</li>
+                            <li>Student Interim sollicite son réseau d’étudiants pour vous ! Nous cherchons et trouvons des candidats qui correspondent à votre besoin.</li>
+                            <li>Nous prenons de 12% à 15% du salaire annuel de l’étudiant recruté.</li>
+                        </ul">
```
Comment (LucyMerlier) [inline templates/home/price_list.html.twig:81]: rogue double quotes in closing ul tag

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by maxime-guichard — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**